install.sh

```diff
@@ -200,6 +200,34 @@ file_not_grpowned() {
   [[ " $(id -G "$USER") " != *" $(get_group "$1") "*  ]]
 }
 
+old_ruby() {
+  if builtin command -v ruby &>/dev/null; then
+    local ruby_version
+    ruby_version=$(ruby <<< 'puts RUBY_VERSION' | grep -o '^[0-9]\+\.[0-9]\+')
```

Comment:
Changed  `^[0-9]*\.[0-9]*` to `^[0-9]\+\.[0-9]\+` to disallow zero-length matches.